pkg/minishift/openshift/service.go

```diff
@@ -72,7 +89,7 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 }
 
 // Get the available routes to user
-func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
+func GetServiceURLs(serviceListNamespace string, ip string) ([]ServiceURL, error) {
```

Comment:
Also _getService_ has wrong godog. There it talk about returning JSON, which I  believe is not quite correct.